config updates

Frazedo opened the pull request:
adjusted m8pv2 cfg motor aliases to use _UART instead of _CS.

printer.cfg updated to move ADXL pins to thb, and add thb mcu temperature sensor.

hartk1213 commented:
@Frazedo looks good if you wanted to update the v1.1 alias file as well that would be awesome then I can get it merged 👍🏼

Frazedo commented:
Done and done.  Also, re-updated the v2 doc as well.  Caught that the pin for E motor was incorrect.

hartk1213 commented:
@Frazedo thank you!

Frazedo commented:
Glad I could help @hartk1213 !  Could you take a look at the printer.cfg file as well?  I added the thb mcu temperature sensor, and fixed the adxl pin calls since they were pointing towards the mcu and not thb.

hartk1213 commented:
@Frazedo yup i saw that , good call on those
